### PR TITLE
[11.0][MIG] analytic

### PR DESCRIPTION
**Pull request by rruebner**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by StefanRijnhart — APPROVED** [review]
Thanks! Waiting until dependency `mail` is merged before merging this one.

**Comment by pedrobaeza** [comment]
Dependency is merged, so I merge this one.

**Pull request merged**
